<launch>
	<node pkg="next_best_view" type="next_best_view_robotmodel_service" name="nbv_srv" output="screen">
	  <rosparam file="$(find next_best_view)/rsc/next_best_view_settings_real.yaml"/>
	</node>
</launch>
